user/inc/application.h

```diff
@@ -52,7 +52,13 @@
 #include "spark_wiring_network.h"
 #include "spark_wiring_client.h"
 #include "spark_wiring_startup.h"
+
```

Comment:
Was this the only way? Was there a way to make the Spark Timer class call into the bluz HAL functions? I looked briefly, I know it goes more through concurrent_hal which plays with the RTOS, but is that route not possible?